src/xrGame/ui/UILoadingScreenHardcoded.h

```diff
@@ -164,34 +164,47 @@ R"(<w>
 inline pcstr GetLoadingScreenXML()
 {
     const bool is_wide = UICore::is_widescreen();
-    if (ShadowOfChernobylMode)
+    const u32 loadingScreenType = pSettingsOpenXRay->read_if_exists<u32>("gameplay", "loading_screen", callOfPripyatMode);
```

Comment:
Checking for numbers in this case is bad, text is preferred... But for this file, it will be easier to just keep things in this file hardcoded, i.e. don't check for openxray.ltx values and check game mode directly. Anyway, in the original engine the loading screen was hardcoded with no ability to change it.

If a modder wants to change the loading screen, he can place modified XML files into gamedata.

Reply:
Okay. I replaced on a string